langchain4j-hugging-face/src/main/java/dev/langchain4j/model/huggingface/util/UrlUtil.java

```diff
@@ -0,0 +1,30 @@
+package dev.langchain4j.model.huggingface.util;
+
+import org.apache.commons.validator.routines.UrlValidator;
+
+/**
+ * Url utils. based on apache-commons lib
+ */
+public class UrlUtil {
+
+    private UrlUtil() {}
+
+    /**
+     * Checks if url is valid
+     * @param url String
+     * @return boolean result
+     */
+    public static boolean isValidUrl(String url) {
```

Comment:
Why do we need to validate the URL? Seems like an overkill TBH

Reply:
It was a suggestion of AI pull request agent ))))

I don't think it's overkill. Just check when class creating. user friendly)

Reply:
Removed.

Reply:
...